### ISSUE TITLE
Foldseek search database

### ISSUE THREAD

**Issue by jmoojun** [steineggerlab/foldseek#394]
Hi. Thank you for making such a nice tool.

I have a question regarding structure search in the AlphaFold training set.
In my test set, I would like to exclude pdbs that are above TM-score=0.9 compared to the whole AlphaFold training set.
I would like to use the cutoff 2018-04-30 when compared to the whole PDB, but the PDB100 database doesn't seem to be the whole PDB.
In the Foldseek search server (search.foldseek.com), is it possible to search against the whole pdb, not PDB100?
What would be other alternative ways to conduct such a search?

**Comment by milot-mirdita** [comment]
The PDB100 is a cluster database, if you use the `--cluster-search 1` parameters it will search against everything, otherwise it will search against the non redundant representatives
